design/FY2021/Enhancement to Simple Window Model.md

```diff
@@ -0,0 +1,166 @@
+
+## Approach ##
+
+The simple window model in EnergyPlus is accessed through the WindowMaterial:SimpleGlazingSystem input object and converts the simple indices into an equivalent single layer window. Once the model generates the properties for the layer, the program reuses the bulk of the layer-by-layer model for subsequent calculations. In this model, user input window U-values include interior and exterior surface heat transfer coefficients. The resistance of the bare window product, or glass-to-glass resistance is augmented by these film coefficients so that,
+
+U =  1/(1/h<sub>(i,w)</sub>  + R<sub>(l,w)</sub>  + 1/h<sub>(o,w)</sub>) &nbsp;&nbsp; Eq. (8)
+
+Where,
+
+h<sub>i,w</sub> is the interior film coefficient under standard winter conditions in units of m<sup>2</sup>·K/W, h<sub>o,w</sub> is the exterior film coefficient under standard winter conditions in units of m<sup>2</sup>·K/W, and R<sub>l, w</sub> is the resistance of the bare window under winter conditions (without the film coefficients) in units of m<sup>2</sup>·K/W. 
+
+In the current implementation, because the window model in EnergyPlus is for flat geometry, the model is not necessarily applicable to low-performance projecting products, such as single-pane windows with uninsulated aluminum frame. The model cannot support glazing systems with a U higher than 7.0 because the thermal resistance of the film coefficients alone can provide this level of performance and none of the various resistances can be negative.
+
+In order to remove this restriction to model glazing systems with high conductive frames, we will introduce the concept of film coefficient adjustment ratio to h<sub>i, w</sub> and h<sub>o, w</sub>. For U input higher than 7.0, the original h<sub>i, w</sub>, origin and h<sub>o, w</sub>, origin will be calculated using U<sub>0</sub> = 7.0, and adjusted film coefficient adjustment ratio α to both inside and outside can be calculated as α= U/U<sub>0</sub>.
+
+αU<sub>0</sub>  =  1/(1/αh<sub>i,w</sub>   + R<sub>l,w</sub>/α  + 1/αh<sub>o,w</sub>) &nbsp;&nbsp; Eq. (9)
+
+Based on Equation (9), during the simulation, when final hin and hout are calculated for window objects using simple window models, these factors will be applied back to adjust hin and hout as αhin and αhout, accordingly. Since the derived R<sub>l,w</sub> is neglectable when U > 7.0, we don’t apply the α back to the window glazing layer.
+We will also add the fields **Inside Face Convective Coefficient Adjustment Ratio** and **Outside Face Convective Coefficient Adjustment Ratio** to **SurfaceProperty:ConvectionCoefficients** and **SurfaceProperty:ConvectionCoefficients:MultipleSurface**. Users are allowed to input the film coefficient adjustment ratios to manually adjust the calculated coefficients.
+
+```
+SurfaceProperty:ConvectionCoefficients,
+  …
+  N10, \field Convective Coefficient 1 Adjustment Ratio
```

Comment:
Why I did not see these added to the .idd file? (at least not showing up in GitHub diffs)?

Reply:
Sorry, this NFP is not up-to-date. The new fields will not be added. I will update the .md file soon.

Reply:
Just try to understand the scope here: a general question I have on this new feature is that what can I use the feature for the following scenario: 

Suppose that you have a window that has a very obtrusive/extrusive frame (say 3-inch wide and 3-inch deep), which will give you a large exposed “fin” side exposure area; and also another ordinary much “flatter” frame structure that does not have much “fin” structure---let’s say an extreme case of just a very flat frame that is pretty much flushed with the center glass surface. How would I use the current feature to model the difference between these two frames/windows? 

If I understand the scope of the current work correctly, the film coefficient calculation not included in EnergyPlus in the current stage (i.e. calculate a more new frame film coefficient base on some geometry inputs, such as the a 3-inch wide and 3-inch deep frame, and/or some other inputs).

Reply:
Hi @jcyuan2020, the NFP is updated now. This feature tries to allow for the simulation of simple-glazing windows with a higher U-factor. The users would distinguish the two cases you mentioned above by the input U value they to the SimpleGlazingWindow object in the idf file. They will not input the detailed frame geometry.